chainer/links/model/vision/googlenet.py

```diff
@@ -255,10 +255,29 @@ def extract(self, images, layers=None, size=(224, 224), **kwargs):
 
            ``train`` and ``volatile`` arguments are not supported anymore since
            v2.
-           Instead, use ``chainer.using_config('train', train)`` and
-           ``chainer.using_config('enable_backprop', not volatile)``
-           respectively.
-           See :func:`chainer.using_config`.
+           Users should configure training and volatile mode
```

Comment:
I think that this issue is not only for users upgrading from v2. The method name `extract` sounds like saying `it does not update parameters at all`.

How about inserting sentence below, before the warning block?

```
Unlike `predict` method, this method does not override `train` and `enable_backprop` configuration.
If you want to extract features without updating model parameters, you need to manually set configuration when calling this method, as follows:

(code example)
```